notebooks/Benchmark_notebook.ipynb

```diff
@@ -274,13 +301,52 @@
     "torchquad.plot_convergence(steps_per_method,values,ground_truth_1d,names)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plotting the total runtime for integration (creation of integration grid + function evaluation + integration computing)."
```

Comment:
I don't think that is true. Because above in `evaluate` you subtracted grid and runtime from integration time (which doesn't actually work at the moment , because you don't measure total evaluation time of function calls atm but only one function  call)